xml/day2_ui_documentation.xml

```diff
@@ -163,6 +163,14 @@
      </imageobject>
     </mediaobject>
    </figure>
+    <note>
+      <para>
+        Packages with the <filename>venv-</filename> prefix denote the version of
+        the specific openstack package that is deployed. The OpenStack releases
+        (<uri>https://releases.openstack.org/</uri>) page can be refrenced to
```

Comment:
Packages with the <literal>venv-</literal> prefix (the filename tag is for files or directories).

For the linked sentence I suggest...
The release name can be determined from the <link xlink:href="https://releases.openstack.org/">OpenStack Releases</link> page.

Reply:
Be nice to see the actual text...
`The release name can be determined from the <link xlink:href="https://releases.openstack.org/">OpenStack Releases</link> page.`

Reply:
and the tags...GitHub is picky. use `literal` instead of `filename`.